firmware/console/binary/tunerstudio.cpp

```diff
@@ -101,7 +101,7 @@ extern persistent_config_container_s persistentState;
  * this is a local copy of the configuration. Any changes to this copy
  * have no effect until this copy is explicitly propagated to the main working copy
  */
-persistent_config_s configWorkingCopy;
+engine_configuration_s engineConfigWorkingCopy;
```

Comment:
Can we move PR comment into codebase to a larger extent?

overall my point is that PR comments are often misplaced code comments. taken to extreme in general PR description could be tiny while whatever we have to say is best place in the code

Reply:
PR comment describes the delta, which is usually only/most interesting at commit time.

Reply:
i believe following could be added into code comments:

"scatter/gather overhead so reads and writes can span both portions"
"this XXX holds a copy while YYY does ZZZ"

Reply:
@rusefillc PR has changed a bit after some discussion on Slack

Reply:
@mck1117 need your help - no brain available today for this PR. Relying on your approval.

Reply:
I want to do some testing in actual hardware, will do tonight.